### PR TITLE
Using named cookies to get a correct redirect

### PR DESCRIPTION
**Pull request by parhedberg**
I think that this will solve the issue in #1425 and #1424 .. This has not been tested extensively, but worked for locally for me when changing the files in SD-card. 

Might be something to work on?

### CODE REVIEW

File: code/components/jomjol_fileserver_ota/server_file.cpp [hunk 1]
```diff
@@ -829,12 +829,6 @@ static esp_err_t delete_post_handler(httpd_req_t *req)
             return ESP_FAIL;
         }
 
-        if (filename == 'wlan.ini') {
```
[the remark this answers is not in the document]
Comment (Slider0007) [reply code/components/jomjol_fileserver_ota/server_file.cpp]: @caco3: I wasn't involved in this. I assume you mean @parhedberg.

[the remark this answers is not in the document]
Comment (caco3) [reply code/components/jomjol_fileserver_ota/server_file.cpp]: ah, right, sorry!

=> @parhedberg  What is the reason you reverted this change?
Will you add it again in a separate pull request?
For me it looks ok!

[the remark this answers is not in the document]
Comment (parhedberg) [reply code/components/jomjol_fileserver_ota/server_file.cpp]: Sorry.. It is me that is struggeling with my branches.. I made a new pull request with only the wlan fix in it...

[the remark this answers is not in the document]
Comment (caco3) [reply code/components/jomjol_fileserver_ota/server_file.cpp]: No problem!

Thanks for your contribution in https://github.com/jomjol/AI-on-the-edge-device/pull/1509 !

**Pull request merged**